tx-source/dash_core.php

```diff
@@ -683,6 +686,7 @@
 $Definition['The %s locale pack does not exist.'] = 'The %s locale pack does not exist.';
 $Definition['The %s table is not in the correct format.'] = 'The %s table is not in the correct format.';
 $Definition['The %s was already approved by %s on %s.'] = 'The %s was already approved by %s on %s.';
+$Definition['Thanks for sticking with us.'] = 'Thanks for sticking with us for';
```

Comment:
I this correct?

Reply:
Yes. You can see the translated string in vanilll/internal#2339. Is there a change you would like me to make? Maybe make the code match.

Reply:
I think we've had the discussion that concatenating strings often does not work for translating languages.